packages/sui-i18n/src/i18n.js

```diff
@@ -120,7 +127,10 @@ export default class Rosetta {
     }
 
     return typeof Intl !== 'undefined'
-      ? new Intl.NumberFormat(culture || this._culture, options).format(number)
+      ? new Intl.NumberFormat(culture || this._culture, {
+          ...options,
+          ...this._extendedIntlNumberOptions
```

Comment:
this way you would always be overriding `options` for each particular `.n` call with the preset `extendedIntlNumberOptions`...
 
perhaps I'm wrong but... how about this?

```suggestion
          ...this._extendedIntlNumberOptions,
          ...options
```

Reply:
I understand that this is precisely what is being sought with this change, to be able to override the current ones if necessary.

Reply:
what I mean is...

![image](https://user-images.githubusercontent.com/18154356/222403976-a398c4c7-edeb-4bec-8baa-d7fed634c7a1.png)

in this case, the option `{useGrouping: false}` wouldn't do anything. Is it ok to override what's being explicitly declared when calling the `.n` method with the _more generic_ `extendedIntlNumberOptions`?

IMHO the more specific option should always have a way to override the _more generic_ one.

Reply:
I agree with Jordi.

Reply:
ok, you've convinced me 😝